command_line/export.py

```diff
@@ -466,6 +481,26 @@ def export_json(params, experiments, reflections):
     )
 
 
+def export_shelx(params, experiments, reflections):
+    """
+    Export data in SHELX HKL format
+
+    :param params: The phil parameters
+    :param experiments: The experiment list
+    :param reflections: The reflection tables
+    """
+
+    _check_input(experiments, reflections, params=params)
+
+    # check for a single intensity choice
+    if len(params.intensity) > 1:
+        raise ValueError("Only 1 intensity option can be exported in this format")
```

Comment:
I think the error message could be slightly more helpful, as this will be raised when running on an integrated datafile with the default options i.e. on `dials.export integrated.* format=shelx` (where the auto option will set `params.intensity=[profile, sum]`).
Perhaps something like "Only 1 intensity option can be exported in this format, please choose a single intensity option e.g. intensity=profile"

Reply:
Thanks. I've added your suggested text.